t/062-count.t

```diff
@@ -241,19 +241,24 @@ n = 10
 
 
 === TEST 11: entries under ngx._reqsock_meta
```

Comment:
Will fix the test title accordingly? We're no longer referencing `ngx._reqsock_meta` in the current test block any more. The same applies to the subsequent tests you edited. Thanks!